tests/integration/defs/perf/disagg/utils/config_loader.py

```
@@ -528,11 +529,67 @@ def _apply_env_overrides(self, config_data: dict, model_name: str) -> dict:
+        Replaces placeholders in custom_config paths.
+
+        Args:
+            config: Configuration dictionary
+        """
+        if "accuracy" not in config or "tasks" not in config["accuracy"]:
+            return
+
+        repo_dir = EnvManager.get_repo_dir()
+
+        # Iterate through all tasks (task names are dynamic)
+        for task_name, task_config in config["accuracy"]["tasks"].items():
```

Ruff (0.14.11)

Loop control variable `task_name` not used within loop body

Rename unused `task_name` to `_task_name`

(B007)